### ISSUE TITLE
[FEATURE] Enable the "tie" parameter

### ISSUE THREAD

**Issue by mschwemer** [TYPO3-Solr/ext-solr#1574]
The "tie" value reflects how each score of the queryfields are "concatenated". The default is "0". That means the highest value will win. A value of "1" means that all scores are summed up. 

It would be nice, if this value could be set via TypoScript configuration and added to the querystring (if set).

**Comment by timohund** [comment]
@mschwemer good idea, do you have the time to work an a patch?

**Comment by mschwemer** [comment]
@timohund Yes, I can work on this feature. [in2code](https://www.in2code.de) will sponsor my worktime.
